src/Tribe/PUE/Checker.php

```diff
@@ -467,68 +467,50 @@ public function ajax_validate_key() {
 		/**
 		 * processes variable substitutions for server-side API message
 		 */
-		private function get_api_message( $info ) {
+		private function get_api_message( $pluginInfo ) {
```

Comment:
We (and [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions)) prefer `snake_case` variables over `camelCase`. Any camelCase vars that you see floating around make us cringe and we change them when it is prudent.